packages/react-components/src/molecules/StatusButton.tsx

```diff
@@ -0,0 +1,250 @@
+/** @jsxImportSource @emotion/react */
+import {
+  useState,
+  useEffect,
+  useRef,
+  ReactNode,
+  MouseEventHandler,
+  ComponentProps,
+} from 'react';
+import { css } from '@emotion/react';
+import { Button, chevronUpIcon, chevronDownIcon } from '..';
+
+import { perRem, mobileScreen, formTargetWidth } from '../pixels';
+
+import {
+  paper,
+  steel,
+  colorWithTransparency,
+  tin,
+  lead,
+  neutral200,
+  info100,
+  info500,
+  warning100,
+  warning500,
+  success100,
+  success500,
+} from '../colors';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  [`@media (max-width: ${mobileScreen.max}px)`]: {
+    width: '100%',
+  },
+});
+
+const menuWrapperStyles = css({
+  position: 'relative',
+  display: 'flex',
+  maxWidth: `${formTargetWidth / perRem}em`,
+});
+
+const menuContainerStyles = css({
+  position: 'absolute',
+  display: 'none',
+  overflow: 'hidden',
+  zIndex: 1,
+  width: `${250 / perRem}em`,
+  top: `${8 / perRem}em`,
+  left: 0,
+  backgroundColor: paper.rgb,
+  border: `1px solid ${steel.rgb}`,
+  boxShadow: `0 2px 6px 0 ${colorWithTransparency(tin, 0.34).rgba}`,
```

Comment:
I don't think we have this boxShadow in the designs. It seems really "flat" on Figma